Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -480,16 +544,38 @@ - (void)finishWithContext:(BPExecutionContext *)context {
             context.finalExitStatus = BPExitStatusAppCrashed;
             NEXT([self proceed]);
             return;
+        case BPExitStatusSimulatorDeleted:
+            self.finalExitStatus = BPExitStatusSimulatorDeleted;
+            self.exitLoop = YES;
+            return;
     }
 
 }
 
+- (void)writeDeviceIDFile {
```

Comment:
if it is just for the sake of ipc, I'd rather build a pipe between `bluepill` and `bp` rather than fork() `bp` process. How do we usually decouple binaries while enabling effective communication between them? @ob @vargon